src/index.js

```diff
@@ -156,6 +158,8 @@ export default function (Alpine) {
 
         event.target.value = value;
         event.target.setSelectionRange(cursorPosition, cursorPosition);
+
+        el.dispatchEvent(new Event("input", { bubbles: true }));
```

Comment:
This actually causes an exception in the console:
Uncaught RangeError: Maximum call stack size exceeded.
    at HTMLInputElement.update (alpinejs-number.js:162:22)